docs/content/en/docs/topics/accessing-services/repair/_index.md

```diff
@@ -88,6 +88,12 @@ Notice the new repair added to the list.
 
 See [Schedule a cluster repair](http://cassandra-reaper.io/docs/usage/schedule/).
 
+### Autoscheduling
+
+It is possible to automatically schedule repairs for all non-system keyspaces in the cluster. Cluster's keyspaces are monitored and any modification (adding or removing) are  detected. Adding new ones will get a schedule created for them and removing the keyspace will instead remove the corresponding repair schedule. 
```

Comment:
One minor nit. I think `Cluster's` should be `Clusters'`.

Reply:
Right, if multiple clusters are managed by one Reaper.

Reply:
How about switching to `With auto-scheduling, it is possible to automatically schedule repaors for all non-system keyspaces in a cluster.  A cluster's keyspaces are monitored and any modifications (additions or removals) are detected.  When a new keyspace is created, a new repair scheduled will be automatically created for that keyspace.  Conversely, when a keyspace is removed, the corresponding repair scheduled will be deleted.`